app/views/interactions/index.html.erb

```diff
@@ -43,7 +43,13 @@
           </p>
         </td>
         <td class='text-center'>
-          <%= link_to(interaction.button_text, edit_link_path(interaction_slug: interaction.slug), class: "btn btn-default btn-s") %>
+          <div class='actions add-vertical-margins col-xs-6 no-gutter'
+               data-module="track-click"
+               data-track-action="editButtonClick"
+               data-track-label="<%= edit_link_path(interaction_slug: interaction.slug) %>"
+               data-track-category="userInteraction:LLM">
+               <%= link_to(interaction.button_text, edit_link_path(interaction_slug: interaction.slug), class: "btn btn-default btn-s js-track") %>
```

Comment:
I've removed the div wrapping. And I've moved the label as a property on the link.